__tests__/app.test.js

```diff
@@ -72,3 +72,71 @@ describe("/api/article/:article_id", () => {
       });
   });
 });
+
+const voteUpdate = {
+  inc_votes: 500,
+};
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("status 200: responds with the updated article", () => {
+    return request(app)
+      .patch("/api/articles/3")
+      .send(voteUpdate)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual({
+          article_id: 3,
+          votes: 500,
+          ...body.article,
```

Comment:
If you are just checking ID and votes, consider objectContaining matcher, then it is more flexible for more keys. Without an odd spread going on.